fp.txt

```diff
@@ -0,0 +1,17 @@
+[Section]
+Name = Adobe Flash Player 29
+Version = 29.0.0.140
+License = Freeware
+Description = This archive contains Flash Player and its plugins for web browsers (NPAPI and PPAPI).
+Size = 44.9 MiB
+Category = 6
+URLSite = https://www.adobe.com/products/flashplayer.html
+URLDownload = http://svn.reactos.org/packages/fp_29.0.0.140.zip
+SHA1 = 1e969604941d21827cca70bda54319776493f4fa
+CDPath = none
+SizeBytes = 47169462
+
+[Section.0415]
+License = Darmowe
+Description = To archiwum zawiera program Flash Player i jego wtyczki do przeglądarek internetowych (NPAPI i PPAPI).
```

Comment:
It's either PPAPI or NPAPI, not both.

Reply:
My package has installers for both plugins.